script/translations/develop.py

```diff
@@ -105,10 +104,7 @@ def run_single(translations, flattened_translations, integration):
         integration_strings, flattened_translations
     )
 
-    if download.DOWNLOAD_DIR.is_dir():
-        rmtree(str(download.DOWNLOAD_DIR))
```

Comment:
The reason this is here, is that files can also be no longer there... (cleanup).

Removing this, will no longer do that and thus can break / influence tests as well.

Removing this bit cannot be right.

Reply:
But isnt the json file being created/overriden anyway? 🤔
Also i dont think it does clean up because this is run for every component, and the last one will still leave its en.json in place

Reply:
does the task works successfully if you run it on your machine as it is on `dev` branch?
I am on macbook air m2 sonoma 14.6.1, and I get that error 100% of times I run it so I believe others should be impacted as well

Reply:
anyhow, i created an issue to discuss options there: https://github.com/home-assistant/core/issues/127886